src/test/java/com/sngular/kloadgen/extractor/SchemaExtractorTest.java

```diff
@@ -99,10 +99,10 @@ void testFlatPropertiesListWithAVRO() throws IOException, RestClientException {
 
     jMeterHelperMockedStatic.when(() -> JMeterHelper.getParsedSchema(Mockito.anyString(), Mockito.any(Properties.class))).thenReturn(parsedSchema);
     jMeterContextServiceMockedStatic.when(() -> JMeterContextService.getContext().getProperties()).thenReturn(properties);
-    Mockito.doNothing().when(avroExtractor).processField(Mockito.any(Field.class), Mockito.anyList(), ArgumentMatchers.eq(true), ArgumentMatchers.eq(false));
+    Mockito.when(avroExtractor.processConfluentParsedSchema(Mockito.any(AvroSchema.class))).thenReturn(new ArrayList<>());
     schemaExtractor.flatPropertiesList("avroSubject");
 
-    Mockito.verify(avroExtractor, Mockito.times(2)).processField(Mockito.any(Field.class), Mockito.anyList(), ArgumentMatchers.eq(true), ArgumentMatchers.eq(false));
+    Mockito.verify(avroExtractor, Mockito.times(1)).processConfluentParsedSchema(Mockito.any(AvroSchema.class));
```

Comment:
Please add a variation in each test where you test each Registry connection to avoid testing only the `processConfluentParsedSchema`

Reply:
Do we have any input file in apicurio format?

Reply:
Looks like you need create your own file for this purpose

Reply:
And I have no idea how that file looks like.
So, I open to suggestions.

Reply:
I have had to disable the new tests due to a cast issue during the excution after a merge: I don't know if the problem is in the code or in the tests.